src/content/app/genome-browser/components/drawer/drawer-views/variant-summary/prepareVariantSummaryData.ts

```diff
@@ -45,16 +41,25 @@ const prepareVariantSummaryData = (variant: Variant) => {
   const variantSummaryData: Record<string, unknown> = {};
 
   addVariantPredictions(variant, variantSummaryData);
+  updateHasPhenotypeAssociations(variant, variantSummaryData);
 
   for (const variantAllele of variant.alleles) {
     addVariantAllelePopulationFrequencyData(variantAllele, variantSummaryData); // iterates over population frequencies
-    addClinicalSignificance(variantAllele, variantSummaryData); // iterates over phenotype assertions
     addVariantAllelePredictions(variantAllele, variantSummaryData); // iterates over prediction results
   }
 
   return variantSummaryData as PreparedVariantSummaryData;
 };
 
+const updateHasPhenotypeAssociations = (
```

Comment:
`updateHas` -> just `add`?

Reply:
Also, notice that there is a loop (lines 46-49 above) specifically for variant alleles, so that you don't have to iterate over same alleles over and over? Wouldn't it make more sense to transform this function such that it accepts a variant allele, alongside a store, and if `store.hasPhenotypeAssociations` if false, but allele has phenotype_assertions, then it sets `store.hasPhenotypeAssociations` to true?

Reply:
✅